ModernWallet/Classes/ViewController/CashOut/CashOutSummaryViewController.swift

```diff
@@ -37,6 +45,10 @@ class CashOutSummaryViewController: UIViewController {
         ibanTitleLabel.text = Localize("cashOut.newDesign.iban")
         bicTitleLabel.text = Localize("cashOut.newDesign.bic")
         accountHolderTitleLabel.text = Localize("cashOut.newDesign.accHolder")
+        accountHolderCountryTitleLabel.text = Localize("cashOut.newDesign.accHolderCountry")
```

Comment:
Please, move all localisations in a dedicated function in ViewController extension.